airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalClient.java

```diff
@@ -213,18 +211,31 @@ void refreshRunningWorkflow() {
     } while (token != null && token.size() > 0);
   }
 
+  /**
+   * Refreshes the cache of running workflows, and returns their names. Currently called by the
+   * Temporal Cloud migrator to generate a list of workflows that should be migrated. After the
+   * Temporal Migration is complete, this could be removed, though it may be handy for a future use
+   * case.
+   */
+  public Set<String> getAllRunningWorkflows() {
+    final var startTime = Instant.now();
+    refreshRunningWorkflow();
+    final var endTime = Instant.now();
+    log.info("getAllRunningWorkflows took {} milliseconds", Duration.between(startTime, endTime).toMillis());
+    return workflowNames;
+  }
+
   public ConnectionManagerWorkflow submitConnectionUpdaterAsync(final UUID connectionId) {
     log.info("Starting the scheduler temporal wf");
     final ConnectionManagerWorkflow connectionManagerWorkflow = ConnectionManagerUtils.startConnectionManagerNoSignal(client, connectionId);
-
+        connectionManagerWorkflow.getState());
```

Comment:
Is this line necessary? It also seems to be strangely indented

Reply:
Oops this was part of a log line that I deleted, but apparently I didn't delete it hard enough